libs/components/modals/src/lib/modules/modal/modal.component.scss

```diff
@@ -141,6 +141,7 @@
 .sky-modal-content > ::ng-deep sky-dock {
   bottom: -$sky-padding-plus-half;
   margin-left: -$sky-margin-plus-half;
+  margin-right: -$sky-margin-plus-half;
```

Comment:
Prevent a horizontal scrollbar when using back to top in a modal.

Reply:
Do we need an overwriting style in the modern theme above like we do for `margin-left`? We clear that by setting `initial` in modern theme.

Reply:
Good catch. Updated.